### PR TITLE
Fix anc registration crash

### PR DESCRIPTION
**Pull request by issyzac**

### CODE REVIEW

File: opensrp-chw/src/kk/assets/json.form/add_anc_registration.json [hunk 1]
```diff
@@ -366,7 +362,7 @@
         "calculation": {
           "rules-engine": {
             "ex-rules": {
-              "rules-file": "add_anc_registration_calculation.yml"
+              "rules-file": "add_anc_registration_relevance.yml"
```
Comment (gosso22) [inline opensrp-chw/src/kk/assets/json.form/add_anc_registration.json:365]: Do you have the calculations in the relevance file?

File: opensrp-chw/src/kk/assets/rule/add_anc_registration_calculation.yml [hunk 2]
```diff
@@ -4,15 +4,15 @@ description: surname calculation
 priority: 1
 condition: "step1_member_consent.contains('member_consent_yes')"
 actions:
-  - "calculation = step1_same_as_fam_name.contains('same_as_fam_name')? step1_fam_name : step1_surname"
+  - "calculation = step1_same_as_fam_name.contains('same_as_fam_name')? step1_fam_name : step1_surname : ''"
```
Comment (gosso22) [inline opensrp-chw/src/kk/assets/rule/add_anc_registration_calculation.yml:7]: Just wanted to make sure that this ternary operator has two colons, what is the expected here?

File: opensrp-chw/src/kk/assets/json.form/add_anc_registration.json (hunk 1 above)
Comment (issyzac) [reply opensrp-chw/src/kk/assets/json.form/add_anc_registration.json:365]: Yes, to improve the performance ya hiyo form, realized with those calculations if you have relevance and calculations on different files the form becomes really slow

File: opensrp-chw/src/kk/assets/rule/add_anc_registration_calculation.yml (hunk 2 above)
Comment (issyzac) [reply opensrp-chw/src/kk/assets/rule/add_anc_registration_calculation.yml:7]: I missed that one, let me correct it and push the changes

**Review by gosso22 — APPROVED** [review]

**Pull request merged**